packages/deployment/Makefile

```diff
@@ -66,4 +66,4 @@ docker-push-solo:
 
 docker-push-deployment:
 	$(DONT_PUSH_LATEST) docker push agoric/deployment:latest
-	docker push $(REPOSITORY)-solo:$(TAG)
```

Comment:
what did this used to do? how was it even close?

Reply:
Fortunately, the old code was never executed (instead the `.github/workflows/docker.yml` built and tagged the images).  I ran into this only because journalbeat failed to compile in the Github action, and I needed to push the images manually.

Which upon inspection wasn't pushing the `agoric/deployment` tag as I expected.  So I fixed it.

Reply:
It was introduced as part of a refactoring to fix a prior refactoring.